pkg/controller/deployment/deployment_controller.go

```diff
@@ -634,3 +658,16 @@ func (dc *DeploymentController) updateDeployment(deployment *extensions.Deployme
 	// TODO: Using client for now, update to use store when it is ready.
 	return dc.expClient.Deployments(deployment.ObjectMeta.Namespace).Update(deployment)
 }
+
+// byCreationTimestamp sorts a list by creation timestamp, using their names as a tie breaker.
+type byCreationTimestamp []*api.ReplicationController
```

Comment:
You can anonymize it and put it in https://github.com/kubernetes/kubernetes/blob/9e66a8f1343f612a25494377df2a8b47ba6d88db/pkg/controller/controller_utils.go